pkg/version/build.go

```diff
@@ -28,8 +28,8 @@ var (
 // Info holds build information
 type Info struct {
 	GitCommit  string `json:"gitCommit"`
-	GitVersion string `json:"GitVersion"`
-	BuildDate  string `json:"BuildDate"`
+	GitVersion string `json:"gitVersion"`
+	BuildDate  string `json:"buildDate"`
 }
```

Comment:
This could be considered a minor breaking change, I don't know why there was a mismatch in the name casing.

Reply:
This JSON output is used in `version` command and `/version` endpoint:
```
$ curl http://localhost:14269/version
{"gitCommit":"","gitVersion":"","buildDate":""}
```
Hard to imaging there would be dependencies on the exact string output of these.

Reply:
I really wouldn't worry about it...